### ISSUE TITLE
Potential security vulnerability in one of your dependencies defined in docs/Gemfile.lock

### ISSUE THREAD

**Issue by praveenkumar** [minishift/minishift#2298]
Msg from minishift repo page.

```
We found a potential security vulnerability in one of your dependencies.

A dependency defined in docs/Gemfile.lock has known security vulnerabilities and should be updated.

Only users who have been granted access to vulnerability alerts for this repository can see this message.
Learn more about vulnerability alerts 
```

**Comment by LalatenduMohanty** [comment]
<img width="777" alt="screen shot 2018-04-16 at 7 19 13 pm" src="https://user-images.githubusercontent.com/3125299/38828041-67992e64-41d2-11e8-90dd-dfa2a6b8df69.png">

**Comment by praveenkumar** [comment]
resolved by #2302
